### DISCUSSION TITLE
PoESP32 configuration

### DISCUSSION THREAD

**Discussion by justbruce** [esphome-econet/esphome-econet#207] [General]
I think i have successfully created a working configuration for the M5Stack PoESP32.   it involves deleting sections from the econet-base.yaml file [anything involving wifi, as ethernet and wifi cannot coexist], so what is the preferred method for integrating this work? Split file ? New fork?

**Comment by barndawgie** [comment]
Can you open an Issue for this please? And include your changes as an example?
